index.html

```diff
@@ -0,0 +1,41 @@
+</<!DOCTYPE html>
+<html lang="en" dir="ltr">
```

Comment:
Why do you have this `dir` attirbute?

Reply:
Because atom put it automatically

Reply:
This is not needed in western countries. You should remove that.